inst/rstudio/templates/project/createTrainingProject.dcf

```diff
@@ -34,8 +34,8 @@ Label: Use Git
 Default: On
 Position: right
 
-Parameter: packrat
-Widget: CheckboxInput
-Label: Use Packrat
-Default: On
-Position: right
+Parameter: packagedeps
+Widget: SelectInput
+Label: Which reproducibility package fields to use
```

Comment:
I would: 

a) shorten this label (e.g. just `Reproducibility`), since a long line messes up the alignment in the template UI
b) group it with the other dropdowns by positioning the check boxes to the right
c) the same applies (both label and alignment) in all the other templates

**Before:**
![image](https://user-images.githubusercontent.com/5055818/30757708-f02471a8-9fcf-11e7-969c-abe6d833fc53.png)

**After:**
![image](https://user-images.githubusercontent.com/5055818/30758237-cc23bf28-9fd1-11e7-819e-ef474fc4f0db.png)

Still looks pretty terrible, but slightly less terrible :smile: